model/riscv_vmem.sail

```diff
@@ -275,8 +286,8 @@ function write_pte forall 'n, 'n in {4, 8} . (
 // Result of address translation
 
 // PUBLIC
-union TR_Result('paddr : Type, 'failure : Type) = {
-  TR_Address : ('paddr, ext_ptw),
+union TR_Result('paddr : Type, 'ext_pte_bits : Type, 'failure : Type) = {
```

Comment:
`ext_pte_bits` is always `extPte` - is there any point parameterising it?

Reply:
please explain more.